docs/source/5-administrator.md

```diff
@@ -60,17 +60,20 @@ These values can be overridden when creating a new FlowDB container by setting t
 
 [^1]:By default, this uses the value set for `cache_size` in `cache.cache_config`.
 
-<a name="logging">
 ## Logging and Audit Trails
 
-FlowKit supports structured logging in JSON form for all interactions with FlowAPI, primarily to facilitate audit trails.
+FlowKit supports structured logging in JSON form throughout FlowMachine and FlowAPI. By default both FlowMachine and FlowAPI will only log errors, and audit logs.
+
+Audit trail logs are _always_ written to both stderr and a log file. Error logs are, by default, only written to stderr. 
```

Comment:
Thanks for changing this to `stderr`. Just to check, does structlog automatically output to stderr? I don't see anywhere in the code that we explicitly configure this (and couldn't find it in the structlog docs at a glance).

Reply:
iirc by default, it goes to stdout, but we're wrapping stdlib's loggers so it goes where they do (stderr).